docs/ldml/tr35-general.html

```diff
@@ -2523,7 +2520,8 @@ <h3>6.<span class='changed'>5</span> <a name="Unit_Sequences" href="#Unit_Sequen
     <p>Units may be used in composed sequences <span  class='changed'>(aka <em>mixed units</em>)</span>, such as <strong>5°
     30′</strong> for 5 degrees 30 minutes, or <strong>3 ft 2
     in.</strong> For that purpose, the appropriate width of the unit
-    listPattern can be used to compose the units in a sequence.</p>
+    listPattern can be used to compose the units in a sequence.<
+    /p>
```

Comment:
Mistake?

Reply:
Hmm, not sure how that happened since I was not working in that area, but just fixed it and squashed.